app/plugin/FeatureInfoWindow.js

```diff
@@ -0,0 +1,142 @@
+                        INFO_FORMAT: 'geojson'
+                    });
+
+                    Ext.Ajax.request({
+                        url: url
+                    }).then(function (response) {
+                        var featureCollection = JSON.parse(response.responseText);
+                        window.add(me.createFeatureCollectionPanel(featureCollection));
+                    });
+                });
+            } else {
+                if (me.highlightSource) {
+                    me.highlightSource.clear();
+                }
+                if (me.window) {
+                    me.window.destroy();
+                }
+            }
+        });
+    },
+
+    showFeatureInfo: function (map, evt) {
+        var me = this;
+
+        if (!this.highlightSource) {
+            this.highlightSource = new ol.source.Vector();
+            new ol.layer.Vector({
+                source: this.highlightSource,
+                style: new ol.style.Style({
+                    image: new ol.style.Circle({
+                        radius: 5,
+                        stroke: new ol.style.Stroke({
+                            color: 'red',
+                            width: 2
+                        })
+                    })
+                }),
+                map: map
+            });
+        } else {
+            this.highlightSource.clear();
+        }
+
+        var feature = new ol.Feature(new ol.geom.Point(evt.coordinate));
+        this.highlightSource.addFeature(feature);
+
+        if (this.window) {
+            this.window.destroy();
+        }
+
+        this.window = Ext.create('Ext.window.Window', {
+            layout: {
+                type: 'accordion',
+                titleCollapse: false,
+                animate: true
+            },
+            width: 300
+        });
+        this.window.show();
+
+        this.window.on('close', function () {
+            me.highlightSource.clear();
+        });
+
+        return this.window;
+    },
+
+    createFeatureCollectionPanel: function (featureCollection) {
+        var me = this;
+
+        return Ext.create('Ext.panel.Panel', {
+            title: featureCollection.name,
+            items: featureCollection.features.map(function (geojsonFeature) {
+                return me.createFeatureGrid(geojsonFeature);
+            })
+        });
+    },
+
+    createFeatureGrid: function (geoJsonFeature) {
+        var props = geoJsonFeature.properties;
+
+        var data = Object.keys(props)
+            .filter(function (key) {
+                return key !== 'geometry';
+            })
+            .map(function (key) {
+                return {
+                    key: key,
+                    value: props[key]
+                };
+            });
+
+        var store = Ext.create('Ext.data.Store', {
+            data: data
+        });
+
+        return Ext.create('Ext.grid.Panel', {
```

Comment:
You can possibly better make use of [`BasiGX.view.grid.FeaturePropertyGrid`](https://github.com/terrestris/BasiGX/blob/master/src/view/grid/FeaturePropertyGrid.js), which has some nice features like clickable href links or attribute mapping, instead of creating a `Ext.grid.Panel` here.

Reply:
Good point.